mycroft/skills/phillips_hue/vocab/en-us/TurnOffKeyword.voc

```diff
@@ -0,0 +1,8 @@
+shut my lights off
+shut off my lights
+shut off the lights
+shut the lights off
+turn my lights off
+turn off my lights
+turn off the lights
+turn the lights off
```

Comment:
Similar what I suggested before, use a `LightKeyword` and `OffKeyword` here to clean up the vocab. The [Volume skill](https://github.com/MycroftAI/mycroft-core/tree/master/mycroft/skills/volume) has good examples.